src/contexts/useUserConsent.tsx

```diff
@@ -91,6 +94,33 @@ export const UserConsentProvider = ({
     [applicantUUID, consents, token, url]
   )
 
+  const addUserConsentStep = useCallback(
+    (steps: StepConfig[]) => {
+      if (
+        !enabled ||
+        !consents ||
+        consents.every(({ required }) => !required)
+      ) {
+        return steps
+      }
+
+      const userConsent: StepConfig = {
+        type: 'userConsent',
+        skip: consents.every((c) => !c.required || (c.required && c.granted)),
+      }
+
+      const welcomeIndex = steps.findIndex(({ type }) => type === 'welcome')
+      const userConsentIndex = welcomeIndex === -1 ? 0 : welcomeIndex + 1
```

Comment:
`-1` looks old fashioned

Reply:
How can I make it trendy?

Reply:
I resolve it for the moment. But we can update coding guidelines or linter later.